test/app/appTest.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from "chai";
```

Comment:
Why is this in a folder called `app`? That doesn't match `src/`'s structure.

Reply:
When I put inside `test/`, the tests weren't ran. But now I realized that I had to tweak the test script, already fixed it. :+1: